### PR TITLE
Delete user schemas from user schemas database only

### PR DESCRIPTION
**Pull request by akshaysonvane**
Fixes https://github.com/marklogic/marklogic-data-hub/issues/1716

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
```
Comment (rjrudin) [inline gradle/wrapper/gradle-wrapper.properties]: Did you mean to update this? If so, I would do it via a separate PR with an explanation of why it's being updated.

File: src/main/groovy/com/marklogic/gradle/task/MarkLogicTask.groovy [hunk 2]
```diff
@@ -44,21 +44,29 @@ class MarkLogicTask extends DefaultTask {
...
+	 * try to connect to the REST API server defined by mlRestPort.
 	 * @return
 	 */
-	DatabaseClient newClient() {
+	DatabaseClient newClient(String database) {
 		if (project.hasProperty("database")) {
 			println "Connecting via the App-Services port to database: " + project.property("database")
 			return getAppConfig().newAppServicesDatabaseClient(project.property("database"))
 		}
+		else if (database != null){
```
Comment (rjrudin) [inline src/main/groovy/com/marklogic/gradle/task/MarkLogicTask.groovy]: I think this should come first - if a client passes in a non-null value, use that. Else, check for the project property. Else, call newDatabaseClient.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (akshaysonvane) [reply gradle/wrapper/gradle-wrapper.properties]: com.github.jk1.dependency-license-report" version "1.3" requires at least grade version 3.3
As discussed, upgraded to gradle wrapper version 4.10.2

File: src/main/groovy/com/marklogic/gradle/task/MarkLogicTask.groovy (hunk 2 above)
Comment (akshaysonvane) [reply src/main/groovy/com/marklogic/gradle/task/MarkLogicTask.groovy]: https://github.com/marklogic-community/ml-gradle/pull/416/commits/cabc11f03c4ef626b4e11eb2e7086b2c6b63ce87 changes the proirity of task property.
